cohort-parent/pom.xml

```diff
@@ -55,6 +58,40 @@
 		<sonar.moduleKey>${project.groupId}:${project.artifactId}</sonar.moduleKey>
 		<sonar.organization>alvearie</sonar.organization>
 		<sonar.host.url>https://sonarcloud.io</sonar.host.url>
+		
+		<!-- Properties added for REST projects-->
+		<whCompanyName>IBM Corp</whCompanyName>
+
+		<whBldVer>${project.version}</whBldVer>
+		<watson.buildtools.version>1.27</watson.buildtools.version>
+
+		<srcjavaDir>src/main/java</srcjavaDir>
+		<srcjavaTestDir>src/test/java</srcjavaTestDir>
+		<srcjavadocDir>${srcjavaDir}</srcjavadocDir>
+		<srcjavadocTestDir>${srcjavaTestDir}</srcjavadocTestDir>
+		<skipTests>true</skipTests>
+		<outDir>${project.build.directory}</outDir>
+		<outDirName>target</outDirName>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
```

Comment:
Lines 75 and 76 are duplicates of 34 and 35?

Reply:
Good catch, will cleanup.